arch/arm64/configs/bcm2711_defconfig

```diff
@@ -1404,6 +1405,7 @@ CONFIG_ANDROID=y
 CONFIG_ANDROID_BINDER_IPC=y
 CONFIG_ANDROID_BINDERFS=y
 CONFIG_NVMEM_RMEM=m
+CONFIG_MUX_GPIO=m
 CONFIG_EXT4_FS=y
 CONFIG_EXT4_FS_POSIX_ACL=y
 CONFIG_EXT4_FS_SECURITY=y
```

Comment:
Some people out there use the arm64 bcmrpi3_defconfig, so let's keep it up-to-date.

Reply:
Dropped from the PR for now.